README.md

```diff
@@ -60,7 +60,7 @@ num_of_public_agents = "1"
 #
 gcp_project = "YOUR_GCP_PROJECT"
 gcp_region = "us-central1"
-gce_ssh_pub_key_file = "/PATH/YOUR_GCP_SSH_PUBLIC_KEY.pub"
+gcp_ssh_pub_key_file = "/PATH/YOUR_GCP_SSH_PUBLIC_KEY.pub"
```

Comment:
I don't think this will work since it's imported from GCP provider. Have you tested this?

Reply:
Yes i have, feel free to look at the file where this is imported from, i think this was a typo.

Reply:
It wasn't. I remember discussing this with @rimusz. Rimas, care to chime in? :D

Reply:
it has to be the `gcp_ssh_pub_key_file` :)

Reply:
Why was it then `gce_...`?